src/cascadia/TerminalApp/App.cpp

```diff
@@ -1067,6 +1117,91 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    winrt::Microsoft::Terminal::TerminalControl::TermControl App::_GetFocusedControl()
+    {
+        int focusedTabIndex = _GetFocusedTabIndex();
+        auto focusedTab = _tabs[focusedTabIndex];
+        return focusedTab->GetFocusedTerminalControl();
+    }
+
+    // Method Description:
+    // - Vertically split the focused pane, and place the given TermControl into
+    //   the newly created pane.
+    // Arguments:
+    // - profile: The profile GUID to associate with the newly created pane. If
+    //   this is nullopt, use the default profile.
+    void App::_SplitVertical(const std::optional<GUID>& profileGuid)
+    {
+        _SplitPane(false, profileGuid);
+    }
+
+    // Method Description:
+    // - Horizontally split the focused pane and place the given TermControl
+    //   into the newly created pane.
+    // Arguments:
+    // - profile: The profile GUID to associate with the newly created pane. If
+    //   this is nullopt, use the default profile.
+    void App::_SplitHorizontal(const std::optional<GUID>& profileGuid)
+    {
+        _SplitPane(true, profileGuid);
+    }
+
+    // Method Description:
+    // - Split the focused pane either horizontally or vertically, and place the
+    //   given TermControl into the newly created pane.
+    // Arguments:
+    // - splitHorizontal: if true, split the pane horizontally. Else split
+    //   vertically.
+    // - profile: The profile GUID to associate with the newly created pane. If
+    //   this is nullopt, use the default profile.
+    void App::_SplitPane(const bool splitHorizontal, const std::optional<GUID>& profileGuid)

```

Comment:
Should the splitHorizontal param be an enum class? The benefits -  the enum will allow for future expansion and it makes the readability of source code more explicit. 

_SplitPane(false, profileGuid);
vs
_SplitPane(Split::Vertical, profileGuid);

Reply:
Yea both you and @dhowett-msft mentioned this, so I'll change it.